doc/data/messages/e/exec-used/details.rst

```diff
@@ -1 +0,0 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
```

Comment:
The available methods and variables used in ``exec()`` may introduce a security hole. You can restrict the use of these variables and methods by passing optional globals and locals parameters (dictionaries) to the  ``exec()`` method.

Reply:
`details.rst` has been created with your text.